tox.ini

```diff
@@ -1,41 +1,40 @@
 [tox]
-minversion = 3.3.0
-isolated_build = true
-isolated_build_env = .package
+minversion = 4.0.0
 
 # https://docs.djangoproject.com/en/dev/faq/install/#what-python-version-can-i-use-with-django
 envlist =
-    py{37,38,39}-django{22,30}-drf{310,311,312},
-    py{37,38,39}-django{31,32}-drf{311,312},
-    py{39,310}-django{40,41}-drf{313,314},
-    py311-django{40,41,42,50,51}-drf315,
-    py312-django{42,50,51}-drf315,
-    py313-django{51}-drf315,
-    py38-{lint, docs},
-    # py313-djmaster  # ModuleNotFoundError: No module named 'psycopg'
+    # Python 3.9: Django <= 4.2
+    py39-django{40,41}-drf{313,314,315},
+    py39-django42-drf{314,315},
+    # Python 3.10 & 3.11: Django <= 5.2
+    py31{0,1}-django{40,41}-drf{313,314,315},
+    py31{0,1}-django{42,50}-drf{314,315},
+    py31{0,1}-django{51,52}-drf315,
+    # Python 3.12: all supported versions & Python 3.13: Django >= 4.1
+    py312-django{40,41}-drf{313,314,315},
+    py313-django41-drf{313,314,315},
+    py31{2,3}-django{42,50}-drf{314,315},
+    py31{2,3}-django{51,52}-drf315,
+    py31{2,3}-prerelease,
```

Comment:
This env list is likely overkill with the simple `tox -l | grep $PYTHON_VERSION` selection that's used for the CI testing, so either we may want to limit the number of Python versions that are part of the CI matrix or we may want to limit the test suite, but I was enumerating all valid versions to start that discussion.